identity-iota/src/did/doc/iota_document.rs

```diff
@@ -404,6 +523,8 @@ impl IotaDocument {
   }
 }
 
+impl<'a, 'b, 'c> IotaDocument {}
```

Comment:
🤔

Reply:
Ummmmm I think this was from something else I was playing around with, plz disregard haha (and removed, of course).